Cubical/Categories/Displayed/Constructions/StructureOverC.agda

```diff
@@ -0,0 +1,58 @@
+-- | Structure displayed over a category.
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Displayed.Constructions.StructureOverC where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Displayed.Functor
+open import Cubical.Categories.Displayed.HLevels
+open import Cubical.Categories.Displayed.TotalCategory.Base
+
+private
+  variable
+    ℓC ℓC' ℓCᴰ ℓCᴰ' ℓD ℓD' ℓDᴰ ℓDᴰ' : Level
+
+record StructureOverC (C : Category ℓC ℓC') ℓCᴰ ℓCᴰ' :
```

Comment:
For naming, I think `StructureOver` is nicer (and same for `PropertyOver`). Then for example you would write `Algebra M : StructureOver Set` for `M : Monad Set`.